src/server.ts

```diff
@@ -0,0 +1,84 @@
+import express, { Application, Request, Response } from "express";
+import querystring from "querystring";
+import request from "request";
+import dotenv from "dotenv";
+dotenv.config();
+
+const APP: Application = express();
```

Comment:
```suggestion
const server: Application = express();
```

`nit`: we should only the all caps pattern when dealing with strings that never change